qiskit/visualization/pulse/matplotlib.py

```diff
@@ -308,7 +310,10 @@ def draw(self, pulse: SamplePulse,
         bbox = ax.get_position()
 
         if self.style.title_font_size > 0:
-            figure.suptitle(str(pulse.name),
+            title = str('')
+            if draw_title:
+                title = str(pulse.name)
+            figure.suptitle(title,
```

Comment:
Should this be indented in the conditional block?

Reply:
Yes,cause no where to call figure.suptitle out of the conditional block